pushkin/database/migrations/versions/61e52d26ebea_message_blacklist.py

```diff
@@ -0,0 +1,37 @@
+"""message_blacklist
+
+Revision ID: 61e52d26ebea
+Revises: 866d344d7b5d
+Create Date: 2016-06-21 11:41:46.919707
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '61e52d26ebea'
+down_revision = '866d344d7b5d'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    context = op.get_context()
+    connection = op.get_bind()
+
+    if not context.dialect.has_table(connection.engine, 'message_blacklist'):
```

Comment:
Remove this if. Table should not exist, because it is not created in previous revisions. If it exist by chance, that is a problem and upgrade should fail.